tools/tidy

```diff
@@ -55,8 +55,8 @@ perltidy_version_expected=$(sed -n "s/^.*Perl::Tidy[^0-9]*\([0-9]*\)['];$/\1/p"
 # This might be used from another repo like os-autoinst-distri-opensuse
 if [ -z "${perltidy_version_expected}" ]; then
     # No cpanfile in the linked repo, use the one from os-autoinst instead
-    dir="$(dirname "$(readlink -f "$0")")"
-    perltidy_version_expected=$(sed -n "s/^.*Perl::Tidy[^0-9]*\([0-9]*\)['];$/\1/p" "$dir"/../cpanfile)
+    cpanfile_dir="$(dirname "$(readlink -f "$0")")"
```

Comment:
So that's an unrelated fix.
I wonder why that problem didn't pop up earlier.

Reply:
> So that's an unrelated fix. 
Yes. Should I move it to separate commit?

> I wonder why that problem didn't pop up earlier.
I have no idea. Complexity of the CI is quite big.

Reply:
> The CI problem with a missing codecovbash popped up recently already, and I don't know what's going wrong there. A rerun usually fixes it.

Thanks for info. openQA CI are complex, long running and error prone.